.travis.yml

```diff
@@ -18,6 +18,10 @@ matrix:
         - php: 7.2
         - php: 7.3
           env: deps=low
+        - php: 7.4
+          env: SYMFONY_PHPUNIT_VERSION=9.4
+        - php: nightly
+          env: SYMFONY_PHPUNIT_VERSION=9.4
```

Comment:
Why PHPUnit 9.4 specifically?

Reply:
I chose 9.4 as it's the most recent right now, so the most susceptible one to have bug fixes.

Requires at least 9.3 as support for PHP 8 was added at that time.